.github/workflows/ci-lint-examples.yml

```diff
@@ -13,8 +13,14 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: 12.x
+    - uses: actions/cache@v1
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-node-linting-examples-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-node-linting-examples-
     - name: Get node modules
-      run: npm ci
+      run: npm i
```

Comment:
Why change to `npm i`? This means that NPM will install in a configuration that does not match the `package-lock.json` and can lead to some really hard to debug bugs.

Reply:
Oh, thanks for the note.